### PR TITLE
SwarmROS updated for building

### PR DESCRIPTION
**Pull request by amilankovich-slab**
[Previously ](https://github.com/ros/rosdistro/pull/23586)SwarmROS did not build successfully, as some dependencies were mission. this is an attempt to fix that issue.

### CODE REVIEW

File: kinetic/distribution.yaml [hunk 1]
```diff
@@ -17572,5 +17573,10 @@ repositories:
       url: https://github.com/zivid/zivid-ros.git
       version: master
     status: maintained
+  swarmros:
```
Comment (tfoote) [inline kinetic/distribution.yaml]: This section should be a subsection of the already existing swarmros entry.

File: kinetic/distribution.yaml [hunk 2]
```diff
@@ -17572,5 +17573,10 @@ repositories:
...
+    doc:
+      type: git
+      url: https://github.com/amilankovich-slab/swarmros-release.git
```
Comment (tfoote) [inline kinetic/distribution.yaml]: The doc url should be a branch (preferred, but tag is ok) on the upstream repository. Usually it's the same as the source entry.

**Review by tfoote — CHANGES_REQUESTED** [review]
I noticed that the package.xml [shows 0.9.3 as the version](https://github.com/amilankovich-slab/swarmros/blob/86179063f45df659f1b5559adb80c17cbad46823/package.xml#L6)  but this PR is for 0.3.1

File: kinetic/distribution.yaml [hunk 3]
```diff
@@ -15029,13 +15029,18 @@ repositories:
...
       test_pull_requests: true
       type: git
       url: https://github.com/amilankovich-slab/swarmros.git
       version: master
     status: developed
     status_description: developed
+    doc:
+      type: git
+      url: https://github.com/amilankovich-slab/swarmros.git
+      version: 0.3.1-4
```
Comment (tfoote) [inline kinetic/distribution.yaml]: This version needs to be a branch (or tag) in the upstream repo. Usually it's set to the same branch as the source entry above so `master`.
```suggestion
      version: master
```

**Review by tfoote — APPROVED** [review]
Thanks for the updates. this looks good to me now.

**Comment by tfoote** [comment]
Sorry one last fixup is that the doc block needs to be at the top due to alphabetical ordering constraints.

**Comment by clalancette** [comment]
Looks good now, but holding for [sync](https://discourse.ros.org/t/preparing-for-kinetic-sync-2020-02-07/12677)

**Pull request merged**